### ISSUE TITLE
Menu issue

### ISSUE THREAD

**Issue by gregorymjason** [rpgboss/rpgboss#480]
Hey guys, got a weird issue that's pretty much going to make my RPG unplayable if I don't solve for this one. Which sucks because I have put a good bit of story into this and would like to see it through.

So when a character uses an item through the menu, or an event removes the item, it creates a blank row in the inventory. Every consumable item after that blank space no longer works when the user is in the menu and attempts to consume them. 

After hacking around in the menu script for a bit I think that's what the organize button should solve for, by either filling in the gaps or allowing me to move an item into a gap thus making everything usable again. While it would be better if the program just moved everything up a spot in the array (my understanding of how the inventory system probably works), or even better, just worked with the gap, I am willing to go with any sort of solution.

Any thoughts?

**Comment by gregorymjason** [comment]
Yeah looking at the last update to the code on this project 12/16, I am going to assume that the creator either ran into an issue that he couldn't solve and makes all RPG's created in this tool unplayable, (like the above) or he just got tired of working on this for free and moved on to a more exciting / lucrative project. I am going to be biting the bullet and using RPG Maker MV, but I appreciate what this tool did for me in terms of allowing me to play and learn the mechanics of an RPG maker without starting a trial...

**Comment by EgyLynx** [comment]
Oh no... :(
